specifyweb/stored_queries/views.py

```diff
@@ -79,6 +79,11 @@ def ephemeral(request):
 
     check_permission_targets(collection.id, request.specify_user.id, [QueryBuilderPt.execute])
     data = run_ephemeral_query(collection, request.specify_user, spquery)
+    for idx in range(len(data['results'])):
```

Comment:
hmm, I'm not sure what this change is about.
I was expecting the change would be made in this file - https://github.com/specify/specify7/blob/production/specifyweb/stored_queries/format.py (to mirror the front-end change)